feedback-module/src/components/Module.js

```diff
@@ -253,11 +261,14 @@ function Module({ pagetitle, endpoint, dir }) {
                     <ModuleButton
                       buttonText={t(text)}
                       className={`flex-button flex-button--${type}`}
-                      networkError={failedRequest}
+                      networkError={
+                        failedRequest && index === screen.buttons.length - 1
```

Comment:
Why are we now checking the index is the length - 1?

Reply:
This is to make sure that the network error alert only displays on the last button when there are multiple on the screen